demo/android/sdl2surface_rawfb/build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# This script is based on SDL sources => androidbuild.sh
+#
+##
+
+SOURCES=()
+MKSOURCES=""
+CURDIR=`pwd -P`
+
+
+if [ -z "$SDLPATH" ];then
+    echo "Please set the SDLPATH directory to the path of the SDL2 source code"
+    exit 1
+fi
+
+if [ -z "$ANDROID_HOME" ];then
+    echo "Please set the ANDROID_HOME directory to the path of the Android SDK"
+    exit 1
+fi
+
+if [ ! -d "$ANDROID_HOME/ndk-bundle" -a -z "$ANDROID_NDK_HOME" ]; then
```

Comment:
Should be `if [ ! -d "$ANDROID_HOME/ndk-bundle" ] && [ -z "$ANDROID_NDK_HOME" ]; then` as more than one comparison will never be supported according to the latest POSIX (yes, we're in bash here, but while I'm at nitpicking... :wink:).

Reply:
done  😉